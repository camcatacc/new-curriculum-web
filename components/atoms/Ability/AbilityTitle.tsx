// Modules
import React from "react";

// Elements
import { Typography } from "@material-ui/core";

// Definitions
export interface AbilityTitleProps {
	children: any;
	style?: React.CSSProperties;
}

// Element
const AbilityTitle = ({ children, style }: AbilityTitleProps) => {
	return (
		<Typography variant="h3" align="center" style={style}>
			{children}
		</Typography>
	);
};

export default AbilityTitle;
