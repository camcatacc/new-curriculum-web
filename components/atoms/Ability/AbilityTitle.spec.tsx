// Modules
import React from "react";
import { render } from "@testing-library/react";

// Elements
import AbilityTitle from "components/atoms/Ability/AbilityTitle";

// Tests
describe("AbilityTitle", () => {
	it("Renders as expected", () => {
		const { container } = render(<AbilityTitle>Hello, hello</AbilityTitle>);
		expect(container).toMatchSnapshot();
	});
});
