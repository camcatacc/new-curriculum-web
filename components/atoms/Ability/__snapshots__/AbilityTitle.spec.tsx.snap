// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AbilityTitle Renders as expected 1`] = `
<div>
  <h3
    class="MuiTypography-root MuiTypography-h3 MuiTypography-alignCenter"
  >
    Hello, hello
  </h3>
</div>
`;
